README.md

```diff
@@ -248,6 +248,7 @@ See https://docs.docker.com/reference/cli/docker/container/run/ for more informa
 Here is an example of how to run a built-in end-to-end automated code repair test, within the container (you can change the `out` directory location or directory name, but you must create that directory before running the command):
 
 ```sh
+#[NOTES] --alerts is invalid syntax for end_to_end_acr.py
```

Comment:
This is not true...end_to_end_acr.py still works with the '--alerts' option.  Please verify if you are still getting an "unrecognized arguments" error using the '--alerts' option with end_to_end_acr.py.

Reply:
I have verified that I am using 7120fbf Pull request #126.
Below in the screenshot I attempted to follow the instructions for "redemption/doc/examples
/simple/README.md" using the most current redemption commit.

It's telling me:
 "usage: end_to_end_acr.py [-h] [--repaired-src OUT_SRC_DIR] [--step-dir STEP_DIR] [-b BASE_DIR] [--in-place] [--repair-includes {true,false}] [--no-patch NO_PATCH] source_file compile_commands alerts
end_to_end_acr.py: 
error: unrecognized arguments: --alerts"

![Screenshot from 2024-06-24 11-34-05](https://github.com/cmu-sei/redemption/assets/79768202/8441b4f4-b870-463c-9ce1-7d488c5f450b)

Reply:
Don:
Thanks for the transcript. It sounds like you are using an old version of end_to_end_acr.py. I note that from your final line:

    usage: end_to_end_acr.py [-h] [--repaired-src OUT_SRC_DIR] [--step-dir STEP_DIR] [-b BASE_DIR] [--in-place] [--repair-includes {true,false}] [--no-patch NO_PATCH] source_file compile_commands alerts
end_to_end_acr.py:
    error: unrecognized arguments: --alerts

If I run end_to_end_acr.py with no arguments, I get a slightly different output:

    usage: end_to_end_acr.py [-h] -a ALERTS [--repaired-src OUT_SRC_DIR] [--step-dir STEP_DIR] [-b BASE_DIR]
                             [--in-place] [--repair-includes {true,false}] [-r RAW_AST_DIR]
                             source_file compile_commands

Note the "-a ALERTS" which appears in my output, but not yours.

I suspect you need to rebuild the 'distrib' Docker image. Or you might be better off working with the 'prereq' image, and sharing the Redemption code between host and container.

I would actually recommend rebuilding both the 'prereq' and 'distrib' Docker images, they have both changed recently.

Reply:
Before you rebuild any containers, you should be able to confirm that the problem is your old 'distrib' container. Try doing your transcript again, but this time, use this as your final 'docker run' command:

    sudo docker run -it --rm -v ${PWD}:/host -w /host -v ${PWD}:/code  docker.cc.cert.org/redemption/prereq  bash

(That is, re-try the demo using the 'prereq' container and sharing the Redemption code on your host, instead of using the 'distrib' container.)